content/docs/releases/release-notes/release-notes-1.18.md

```diff
@@ -85,18 +85,43 @@ And finally, thanks to the cert-manager steering committee for their feedback in
 
 ## `v1.18.0`
 
+Changes since `v1.17.0`:
+
 ### Feature
 
-TODO
+- Add config to the Vault issuer to allow the server-name to be specified when validating the certificates the Vault server presents. ([`#7663`](https://github.com/cert-manager/cert-manager/pull/7663), [`@ThatsMrTalbot`](https://github.com/ThatsMrTalbot))
+- Added `app.kubernetes.io/managed-by: cert-manager` label to the created Let's Encrypt account keys ([`#7577`](https://github.com/cert-manager/cert-manager/pull/7577), [`@terinjokes`](https://github.com/terinjokes))
+- Added certificate issuance and expiration time metrics (`certmanager_certificate_not_before_timestamp_seconds`, `certmanager_certificate_not_after_timestamp_seconds`). ([`#7612`](https://github.com/cert-manager/cert-manager/pull/7612), [`@solidDoWant`](https://github.com/solidDoWant))
+- Added ingress-shim option:
+  - --extra-certificate-annotations which sets list of annotation keys to be copied from Ingress-like to resulting Certificate object ([`#7083`](https://github.com/cert-manager/cert-manager/pull/7083), [`@k0da`](https://github.com/k0da))
```

Comment:
```suggestion
  - `--extra-certificate-annotations` which sets list of annotation keys to be copied from Ingress-like to resulting Certificate object ([`#7083`](https://github.com/cert-manager/cert-manager/pull/7083), [`@k0da`](https://github.com/k0da))
```

Reply:
Thanks, I've updated the PR release-note: https://github.com/cert-manager/cert-manager/pull/7083